p2p/pex/reactor.go

```diff
@@ -11,12 +11,24 @@ import (
 	tmmath "github.com/tendermint/tendermint/libs/math"
 	"github.com/tendermint/tendermint/libs/service"
 	"github.com/tendermint/tendermint/p2p"
+	"github.com/tendermint/tendermint/p2p/conn"
 	protop2p "github.com/tendermint/tendermint/proto/tendermint/p2p"
 )
 
 var (
 	_ service.Service = (*ReactorV2)(nil)
 	_ p2p.Wrapper     = (*protop2p.PexMessage)(nil)
+
+	ChannelDescriptors = []*conn.ChannelDescriptor{
```

Comment:
Yeah I agree. I was kind of uncertain about this. Perhaps in the future we want to make a `channels.go` file in node and declare all the channels there since channels are really a node level implementation detail

Reply:
> descriptors into the constructor for the reactor is "cleaner"

The reactors themselves don't need channel descriptor info AFAIK. It's just for the router / transport / switch